code/modules/awaymissions/super_secret_room.dm

```diff
@@ -45,7 +47,7 @@
 			SpeakPeace(list("Alright maybe that's <b>too</b> boring.", "I can't keep manually typing these lines out though.", "It's hard to explain but the code structure I'm using is kind of terrible."))
 		if(10)
 			SpeakPeace(list("Oh I have an idea!", "Lets outsource this endless banter to Poly!", "Then you'll be able to keep listening to this without getting bored!"))
-			if(isnull(shenanigans))
+			if(isnull(shenanigans) || !shenanigans.len)
 				shenanigans = list("Except the poly file is missing...")
 		if(11 to 14, 16 to 50, 52 to 99, 103 to 107, 109 to 203, 205 to 249, 252 to 665, 667 to 999, 1001 to 5642)
 			SpeakPeace(list(pick(shenanigans),pick(shenanigans),pick(shenanigans)))
```

Comment:
While we're editing this file, could you make the line RIGHT below this one more sane?
`if (times_spoken_to % 10 == 0)` or something like that?

Reply:
Not sure what you mean here

Reply:
To change https://github.com/tgstation/tgstation/blob/5a5f9c289110fb52077dcafe939fe7d15763d82d/code/modules/awaymissions/super_secret_room.dm#L52
to `if(times_spoken_to % 10 == 0)` (which is actually slightly different behavior but that's fine)

Reply:
yo